### PR TITLE
Add elevation calculator to simulator

### PR DESCRIPTION
**Pull request by pkubanek**

### CODE REVIEW

File: python/lsst/ts/criopy/m1m3/force_calculator.py [hunk 1]
```diff
@@ -324,10 +353,16 @@ def __add__(self, obj2: Any) -> Self:
             return NotImplemented
 
     def __init__(self, config_dir: None | str | pathlib.Path = None):
+        self._fam_computation: list[pd.DataFrame] = []
+        self._acceleration_computation: list[pd.DataFrame] = []
+        self._velocity_computation: list[pd.DataFrame] = []
```
Comment (gmegh) [inline python/lsst/ts/criopy/m1m3/force_calculator.py:358]: Why do you need these three lines here? They look unrelated to this PR

**Review by gmegh — APPROVED** [review]
LGTM, please see my comment

File: python/lsst/ts/criopy/m1m3/force_calculator.py (hunk 1 above)
Comment (pkubanek) [reply python/lsst/ts/criopy/m1m3/force_calculator.py]: It was reverted, as it was really not needed.
